### ISSUE TITLE
Find and replace in project - is it possible to replace only the selected items

### ISSUE THREAD

**Issue by mtomas7** [atom/find-and-replace#696]
Hello, feature works for me when I want to replace all found instances. But I could not find a way to approve selected instances - clicking or double-clicking on search result lines just opens the particular file with the search text selected.

I'm not really complaining, just thought it could be a useful improvement.

Thank you!

**Comment by thehans** [comment]
Agreed, I'm finding it very tedious to replace particular results(Replace All is not what I want).  I think there should be a plain "replace" button next to "replace all" that would replace the current highlighted instance in the search results, and move the cursor to the next one.  Navigating into each file to change the instances I want is taking very long.

**Comment by mtomas7** [comment]
Or the could be "check" icons next to search instances that could be selected.

**Comment by bxt** [comment]
@mtomas7 Your idea with checkboxes was also suggested in #850. I think this would be a very neat feature and it is pretty much required to make the project search and replace useful in many cases.
